src/PhlyRestfully/ResourceController.php

```diff
@@ -389,7 +389,7 @@ public function delete($id)
         return $response;
     }
 
-    public function deleteList()
+    public function deleteList($data = [])
```

Comment:
This needs to be a required parameter, as zend-mvc made it required in 2.2 or 2.3, IIRC. I'll write a test to check, and, of so, update it during merge.